COMETwebapp/Components/Viewer/ViewerBody.razor.cs

```diff
@@ -124,7 +124,7 @@ private void UpdateUrl()
                 additionalParameters["option"] = this.ViewModel.OptionSelector.SelectedOption.Iid.ToString();
             }
 
-            this.UpdateUrlWithParameters(additionalParameters, nameof(Pages.Viewer.Viewer));
+            this.UpdateUrlWithParameters(additionalParameters, "3DViewer");

```

Comment:
I think nameof() is always better in these cases. or use constants, or enumerations for these kind of important strings.